src/traces/scattergl/attributes.js

```diff
@@ -84,5 +84,17 @@ module.exports = {
     fillcolor: scatterAttrs.fillcolor,
 
     error_y: scatterAttrs.error_y,
-    error_x: scatterAttrs.error_x
+    error_x: scatterAttrs.error_x,
+
+    hoveron: {
```

Comment:
This has no effect in gl2d. You must have added this in to make other things not break, correct?